wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,589 @@
+- There will be 4 No Mod, 1 Hidden, 1 Hardrock and 1 Double Time map in each round
+- Your scores on each map will be added together giving you a total score
+- The top 64 players will progress from the first qualifier
+- The top 32 players will progress from the second qualifier
+- Seeding for groups will be determined from your seed based on your osu! rank (as of signup close) and your overall position in the RO64 qualifer, eg 3rd seed + 15th in qualifier = 18 points
+
+### Rolling
+- When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC Roll System](https://i.imgur.com/S0VIvni.png).
+### Banning
+- From the Group Stage onwards players will have **TWO** bans.
+- Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.
+### Warmups
+- There will be no warmups in Qualification rounds. After this, warmup maps must be a submitted beatmap, and have a drain time of 4 minutes or less
+### Picking
+- Players have **2 minutes** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
+- For Freemod picks, players must select any combination of the following mods: **Hidden, HardRock**
+- **NEW** - Failed scores WILL count
+- Tiebreakers are Freemod, however No Mod can be used.
+
+## Tournament Schedule
+
+![Schedules](img/schedules.png)
+
+| Event | Timestamp |
+| --: | :--|
+| Signups Open | 2019-07-27 |
+| Signups Closed | 2019-08-26 |
+| Qualifying Round 1 | 2019-09-07/2019-09-08 |
+| Qualifying Round 2 | 2019-09-14/2019-09-15 |
+| Round of 32 | 2019-09-21/2019-09-22 |
+| Round of 16 + Losers Round 1 | 2019-09-28/2019-09-29 |
+| Quarter Finals + Losers Round 2/3 | 2019-10-5/2019-10-6 |
+| Semi Finals + Losers Round 4/5 | 2019-10-12/2019-10-13 |
+| Winners Finals + Losers 6/Semi Final | 2019-10-19/2019-10-20 |
+| Losers Final + GRAND Finals | 2019-10-26/2019-10-27 |
+
+
+
+## Prizes
+
+![Prizes](img/prizes.png)
+
+| Placing | Prize(s)|
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "Winner") | 6 months of osu! supporter, a shiny profile badge and a magnificent trophy! |
+| ![Silver Crown](/wiki/shared/crown-silver.png "Second place") | 3 months of osu! supporter and a sparkling medal |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "Third place") | 1 month of osu supporter and a sparkling medal |
+
+### Badge
+![UKCC 7 Winner](img/badge.png)
+
+
+## Organisation
+
+![Staff List](img/staff.png)
+
+UK is run by many wonderful people, this list doesn't fully represent how many amazing people contribute as we accept people along the way. If you're interested in helping staff future UK Community Cups, please contact a host in the [Discord](https://discord.gg/jYhjTtj)!
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host(s) | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
```

Comment:
```suggestion
| Host | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
```
or preferably Manager

Reply:
There's multiple hosts - not sure I agree with this one 🤔